src/chrome/content/rules/Intel_Fellowships.com.xml

```diff
@@ -1,18 +0,0 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://intelfellowships.com/ => https://intelfellowships.com/: (6, 'Could not resolve host: intelfellowships.com')
-Fetch error: http://www.intelfellowships.com/ => https://www.intelfellowships.com/: (6, 'Could not resolve host: www.intelfellowships.com')
-
-	For other Intel coverage, see Intel.xml.
```

Comment:
@J0WI I had to do it in a separate PR since I have deleted the repository: #10959